### PR TITLE
Nova 5 upgrade

### PR DESCRIPTION
**Pull request by christian-heiko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Laravel Nova 4 Roles & Permissions
+# Laravel Nova 5 Roles & Permissions
```
Comment (leandrogehlen) [inline README.md:1]: Maybe remove `5`

File: composer.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "sereny/nova-permissions",
-    "description": "Laravel Nova 4 Grouped Permissions",
+    "description": "Laravel Nova 5 Grouped Permissions",
```
Comment (leandrogehlen) [inline composer.json:3]: Remove `5`

**Review by leandrogehlen — COMMENTED** [review]
Please add changelog entry

**Comment by vbezruchkin** [comment]
hi, thanks for the PR. Please accept it. thanks!

**Comment by leandrogehlen** [comment]
I'm waiting for the review solicitations to be resolved.

**Comment by leandrogehlen** [comment]
@vbezruchkin, i do not have license for Laravel Nova 5.
I have approved the PR, but it's necessary build resource files.

**Pull request merged**